dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -227,7 +235,7 @@ netty-transport-native-kqueue/4.1.115.Final/osx-x86_64/netty-transport-native-kq
 netty-transport-native-unix-common/4.1.115.Final//netty-transport-native-unix-common-4.1.115.Final.jar
 netty-transport/4.1.115.Final//netty-transport-4.1.115.Final.jar
 objenesis/3.3//objenesis-3.3.jar
-okhttp/3.12.12//okhttp-3.12.12.jar
+okhttp/4.12.0//okhttp-4.12.0.jar
```

Comment:
The K8s client "Switch default HTTP client from OkHttp to Vert.x" https://github.com/fabric8io/kubernetes-client/issues/6470

instead of upgrading OkHttp4 and pulling Kotlin runtime deps, maybe following the upstream to switch to Vert.x is a better option?

Reply:
Fine to me. From `6.x` to `7.x`, we need to make some changes first, let's run GA first.

Reply:
I'm +1 for @pan3793 's comment.